pandas/tests/indexing/test_coercion.py

```diff
@@ -1061,23 +1059,16 @@ def test_replace_series_datetime_tz(self):
 
         tm.assert_series_equal(result, exp)
 
-    # TODO(jreback) commented out to only have a single xfail printed
-    @pytest.mark.xfail(
-        reason="different tz, currently mask_missing raises SystemError", strict=False
+    @pytest.mark.parametrize("how", ["dict", "series"])
+    @pytest.mark.parametrize(
+        "to_key",
+        ["datetime64[ns]", "datetime64[ns, UTC]", "datetime64[ns, US/Eastern]"],
     )
-    # @pytest.mark.parametrize('how', ['dict', 'series'])
-    # @pytest.mark.parametrize('to_key', [
-    #    'datetime64[ns]', 'datetime64[ns, UTC]',
-    #    'datetime64[ns, US/Eastern]'])
-    # @pytest.mark.parametrize('from_key', [
-    #    'datetime64[ns]', 'datetime64[ns, UTC]',
-    #    'datetime64[ns, US/Eastern]'])
-    # def test_replace_series_datetime_datetime(self, how, to_key, from_key):
-    def test_replace_series_datetime_datetime(self):
-        how = "dict"
-        to_key = "datetime64[ns]"
-        from_key = "datetime64[ns]"
-
+    @pytest.mark.parametrize(
+        "from_key",
+        ["datetime64[ns]", "datetime64[ns, UTC]", "datetime64[ns, US/Eastern]"],
+    )
+    def test_replace_series_datetime_datetime(self, how, to_key, from_key):
         index = pd.Index([3, 4], name="xxx")
```

Comment:
I know this isn't in the PR diff, but can we change the `name` attribute here.  I know it's all lowercase, but still...

Reply:
changed to "xyz"